webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -107,13 +112,19 @@ class ManageManifestModal extends Component {
       canDeleteManifest,
       canEditOrganizations,
       simpleContentAccess,
+      enableSimpleContentAccess,
+      disableSimpleContentAccess,
     } = this.props;
 
+    const { actionInProgress } = this.state;
+
     const showRedHatProviderDetails = canEditOrganizations;
     const showSubscriptionManifest = (canImportManifest || canDeleteManifest);
     const showManifestTab = (showRedHatProviderDetails || showSubscriptionManifest);
-
-    const { actionInProgress } = this.state;
+    const disableSCASwitch = (disableManifestActions ||
+      !manifestExists(organization) ||
```

Comment:
Please rebase on latest master and update this to use the new Redux selector `selectIsManifestImported` from `OrganizationSelectors.js` (see [comment](https://github.com/Katello/katello/pull/8810#discussion_r453858147) on #8810) instead of `manifestExists`